ConditionalConstructions/Program.cs

```diff
@@ -0,0 +1,390 @@
+﻿using System;
+
+namespace ConditionalConstructions
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Exercise 1");
+            
+            // First Task
+            Console.WriteLine("Task 1");
+            Console.WriteLine("Enter the number");
+            int firstTaskNumber = Convert.ToInt32(Console.ReadLine());
+
+            int evenOrNot = firstTaskNumber % 2;
+            if (evenOrNot == 0)
+            {
+                Console.WriteLine("This number is even");
+            }
+            else if (evenOrNot == 1)
+            {
+                Console.WriteLine("This number is not even");
+            }
+            else
+            {
+                Console.WriteLine("Invalid value entered");
+            }
+            Console.WriteLine("");
+
+            // Second Task
+            Console.WriteLine("Task 2");
+            Console.WriteLine("Enter the mounth number");
+
+            int MounthNumber = Convert.ToInt32(Console.ReadLine());
+
+            if (MounthNumber >= 1 && MounthNumber <= 2)
+            {
+                Console.WriteLine("Winter");
+            }
+            else if ( MounthNumber >= 3 && MounthNumber <= 5)
+            {
+                Console.WriteLine("Spring");
+            }
+            else if (MounthNumber >= 6 && MounthNumber <= 8 )
+            {
+                Console.WriteLine("Summer");
+            }
+            else if (MounthNumber >= 9 && MounthNumber <= 11)
+            {
+                Console.WriteLine("Autumn");
+            }
+            else if (MounthNumber == 12)
+            {
+                Console.WriteLine("Winter");
+            }
+            else
+            {
+                Console.WriteLine("Invalid number");
+            }
+            Console.WriteLine("");
+
+            // Third Task
+            Console.WriteLine("Task 3");
+            Console.WriteLine("Enter number a");
+            int thirdTaskNumA = Convert.ToInt32(Console.ReadLine());
+
+            Console.WriteLine("Enter number b");
+            int thirdTaskNumB = Convert.ToInt32(Console.ReadLine());
+
+            Console.WriteLine("Enter number c");
+            int thirdTaskNumC = Convert.ToInt32(Console.ReadLine());
+
+            if (thirdTaskNumA < thirdTaskNumB && thirdTaskNumB < thirdTaskNumC )
```

Comment:
Console.WriteLine("Expression a < b < c is " + (thirdTaskNumA < thirdTaskNumB && thirdTaskNumB < thirdTaskNumC) );